website/common/locales/en/groups.json

```diff
@@ -266,9 +266,9 @@
   "userRequestsApproval": "<%= userName %> requests approval",
   "userCountRequestsApproval": "<%= userCount %> members request approval",
   "youAreRequestingApproval": "You are requesting approval",
-  "chatPrivilegesRevoked": "You cannot do that because your chat privileges have been revoked.",
-  "cannotCreatePublicGuildWhenMuted": "You cannot create a public guild because your chat privileges have been revoked.",
-  "cannotInviteWhenMuted": "You cannot invite anyone to a guild or party because your chat privileges have been revoked.",
+  "chatPrivilegesRevoked": "You cannot do that because your chat privileges have been removed. For details or to ask if your privileges can be returned, please email our Community Manager at admin@habitica.com or ask your parent or guardian to email them. Please include your @Username in the email. If a moderator has already told you that your chat ban is temporary, you do not need to send an email.",
```

Comment:
this new part here seems common to several strings, could it be moved into its own string?

Reply:
I've replaced the near-duplicate messages with the one generic one.

That was actually the original suggestion I made to the other mods and they were fine with it so this is okay from a moderator perspective.

Ready for review again.